src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -501,47 +501,83 @@ fx_ver_t fx_muxer_t::resolve_framework_version(const std::vector<fx_ver_t>& vers
     const pal::string_t& fx_ver,
     const fx_ver_t& specified,
     const bool& patch_roll_fwd,
-    const int& roll_fwd_on_no_candidate_fx)
+    const enum roll_fwd_on_no_candidate_fx_option& roll_fwd_on_no_candidate_fx)
```

Comment:
>&  [](start = 49, length = 2)

It's consistent with the bool just above, but it's kind of odd (IMHO) pass an enum by reference.

Reply:
I wasn't sure if this was a new convention or not, but I agree. I'll at least change this signature to pass primitives by value